tests/test_create_scitoken.py

```diff
@@ -419,6 +420,29 @@ def test_discover(self):
         if os.path.isfile(bt_tmp):
             shutil.move(bt_tmp, bt_path)
 
+    def test_noiss(self):
+        """
+        Verify that tokens without the `iss` are not accepted.
+        """
+        encoded_jwt = jwt.encode(
+            {
+                "ver": "scitoken:2.0",
+                "aud": "https://demo.scitokens.org",
+                "exp": 1668462237,
+                "iat": 1668461637,
+                "nbf": 1668461637,
```

Comment:
You are hardcoding these values.  Need them to be dynamic.
```suggestion
                "exp": datetime.datetime.now(tz=timezone.utc) + datetime.timedelta(seconds=30),
                "iat": datetime.datetime.now(tz=timezone.utc),
                "nbf": datetime.datetime.now(tz=timezone.utc),
```